plugins/modules/proxmox_backup.py

```diff
@@ -0,0 +1,632 @@
+      - Needs to be a single line, newline and backslash need to be escaped as `\n` and `\\` respectively.
+    default: '{{guestname}}'
+    type: str
+  fleecing:
+    description:
+      - Enable backup fleecing. Works only for virtual machines and their disks.
+      - Must be entered as a string, containing key-value pairs in a list.
+    type: str
+  mode:
+    description:
+      - Specifices the mode to select backup targets.
+    choices: ['include', 'all', 'pool']
+    required: true
+    type: str
+  node:
+    description:
+      - Only execute the backup job for the given node.
+      - This option is usually used if O(mode=all).
+      - If you specify a node id and your vmids or pool do not reside there, they will not be backed up!
+    type: str
+  notification_mode:
+    description:
+      - Determine which notification system to use.
+    type: str
+    choices: ['auto','legacy-sendmail', 'notification-system']
+    default: auto
+  performance_tweaks:
+    description:
+      - Enable other performance-related settings.
+      - Must be entered as a string, containing comma separated key-value pairs.
+      - "For example: max-workers=2,pbs-entries-max=2."
+    type: str
+  pool:
+    description:
+      - Specify a pool name to limit backups to guests to the given pool.
+      - Required, when O(mode=pool).
+      - Also required, when your user only has VM.Backup permission for this single pool.
+    type: str
+  protected:
+    description:
+      - Marks backups as protected.
+      - >
+        "Might fail, when the PBS backend has verify enabled
+         due to this bug: U(https://bugzilla.proxmox.com/show_bug.cgi?id=4289)"
+    type: bool
+  retention:
+    description:
+      - >
+        Use custom retention options instead of those from the default cluster
+        configuration (which is usually 'keep-all').
+      - Always requires Datastore.Allocate permission at the storage endpoint.
+      - >
+        Specifying a retention time other than V(keep-all=1) might trigger pruning on the datastore,
+        if an existing backup should be deleted target due to your specified timeframe.
+      - Deleting requires Datastore.Modify or Datastore.Prune permissions on the backup storage.
+    type: str
+  storage:
+    description:
+      - Store the backup archive on this storage.
+    type: str
+    required: true
+  vmids:
+    description:
+      - The instance ids to be backed up.
+      - Only valid, if O(mode=include).
+    type: list
+    elements: int
+  wait:
+    description:
+      - Wait for the backup to be finished.
+    type: bool
+    default: false
+  wait_timeout:
+    description:
+      - Seconds to wait for the backup to be finished.
+      - Will only be evaluated, if O(wait=true).
+    type: int
+    default: 10
+notes:
+  - Requires proxmoxer and requests modules on host. These modules can be installed with pip.
```

Comment:
```suggestion
```
That's already said implicitly through `requirements`, I don't think it needs to be repeated.